examples/extensions/logger.middleware.ts

```diff
@@ -0,0 +1,55 @@
+import { Service } from "typedi";
+import { GraphQLResolveInfo, GraphQLFieldConfig, GraphQLObjectTypeConfig } from "graphql";
+
+import { MiddlewareInterface, NextFn, ResolverData } from "../../src";
+
+import { Context } from "./context.interface";
+import { Logger } from "./logger.service";
+
+const extractFieldConfig = (info: GraphQLResolveInfo): GraphQLFieldConfig<any, any> => {
+  const { type, extensions, description, deprecationReason } = info.parentType.getFields()[
+    info.fieldName
+  ];
+
+  return {
+    type,
+    description,
+    extensions,
+    deprecationReason,
+  };
+};
+
+const extractParentConfig = (info: GraphQLResolveInfo): GraphQLObjectTypeConfig<any, any> =>
+  info.parentType.toConfig();
+
+const extractLoggerExtensionsFromConfig = (
+  config: GraphQLObjectTypeConfig<any, any> | GraphQLFieldConfig<any, any>,
+) => (config.extensions && config.extensions.log) || {};
+
+const getLoggerExtensions = (info: GraphQLResolveInfo) => {
+  const fieldConfig = extractFieldConfig(info);
+  const fieldLoggernExtensions = extractLoggerExtensionsFromConfig(fieldConfig);
```

Comment:
`Loggern` - typo

Reply:
🤦‍♂ sorry for this, fixed here: 546fadb4de3d51b26f0d55599cac42cca6ea2a81